components/resizable-iframe/index.js

```diff
@@ -0,0 +1,193 @@
+/**
+ * Imported from Calypso, with some lint fixes and gutenburg specific changes.
+ */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import debugFactory from 'debug';
+import { omit } from 'lodash';
+import uuid from 'uuid/v4';
+
+/**
+ * Globals
+ */
+const debug = debugFactory( 'gutenburg:resizable-iframe' ),
+	noop = () => {};
+
+export default React.createClass( {
```

Comment:
`React.createClass` is deprecated and this needs to be rewritten as a `wp.element.Component` class.